src/goMain.ts

```diff
@@ -266,6 +266,10 @@ export function activate(ctx: vscode.ExtensionContext): void {
 		showTestOutput();
 	}));
 
+	ctx.subscriptions.push(vscode.commands.registerCommand('go.test.cancel', () => {
```

Comment:
We should expose this as a command as well. That would help users who are more inclined to use their keyboards and want to avoid the mouse

Reply:
I've pushed a commit to address this

Reply:
Thanks!